kong/runloop/wasm.lua

```diff
@@ -945,13 +945,6 @@ function _M.attach(ctx)
       return kong.response.error(500)
     end
   end
-
-  jit.off(proxy_wasm.start)
-  ok, err = proxy_wasm.start()
-  if not ok then
-    log(CRIT, "failed to execute ", chain.label, " filter chain for request: ", err)
-    return kong.response.error(500)
-  end
 end
```

Comment:
Paging @zhongweiy.

From https://github.com/Kong/ngx_wasm_module/pull/546, you'll see that `proxy_wasm.start()` is removed and no longer used, so we're no longer manually disabling jit here.

When you have time, can you look at the code changes and advise if we need to disable jit anywhere else instead (perhaps `proxy_wasm.attach()`)? I'm curious if the conditions of a796ac6105b60fbd0e85c24281b9f91cf19bdeaf still apply somewhere.

Reply:
Once there is no `ffi call (possible to be compiled)` -> `c function` -> `lua callback function` call chain, it is free from the previous `bad callback` error.

It looks there is no such `ffi call` entry call any more in current design. If yes, I think it is OK to not do `jit.off`.